### DISCUSSION TITLE
Security questions from a client

### DISCUSSION THREAD

**Discussion by huseyincob** [renovatebot/renovate#13399] [Suggest an Idea]
Hello there,

It's Huseyin from Adjust GmbH. We are currently planning to use Renovate, accordingly, I already shared our thoughts with your sales team. In addition to this, we need to have a look at your certifications. Could you provide me with the recent certificates and reports (ISO and SOC2) that you currently have? Also, it'd be great to have a look at several security policies of yours. This is one of the parts of our security evaluation process.

I've tried to get in touch with support but I got redirected to here interestingly.

Thanks!

**Comment by rarkins** [comment]
Renovate is an open source product and as such has no certifications like you're asking. Assuming you're referring to WhiteSource support, they can put you in touch with a sale team to answer questions about commercial products if you are interested in becoming a customer of commercial products. But there's no concept of certifications or commercial support for products you're not paying for.

**Comment by huseyincob** [comment]
I got redirected by WhiteSource support to get to know more about WhiteSource's security posture and security certifications that they have. Thanks, I'll try to reach out them once more.

**Reply by rarkins** [reply]
Were you inquiring about trialing commercial products or asking about the free Renovate offering?

**Reply by huseyincob** [reply]
I'm from the security team of Adjust, some of our internal teams are interested to use several White Source products. WhiteSource's sales team must have more context about it as I'm only interested in security matters.

**Reply by rarkins** [reply]
I suggest you ask your colleagues who are talking to WS sales to put you in touch with the contacts they have, instead of emailing WS support, who mostly deal with post-sales questions. For the non-commercial Renovate offerings, they are correct to redirect you here.
